application-blog-ui/src/main/resources/Blog/BlogStyle.xml

```diff
@@ -226,6 +226,7 @@
   float: right;
   position: relative;
   z-index: 2;
+  margin-left: 1.5em;
```

Comment:
Yeah! you were right. If second line is too long then it comes below blog-entry-button.
Added margin-bottom too.  :)